From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: byquanton <32410361+byquanton@users.noreply.github.com>
Date: Mon, 2 Oct 2023 11:43:24 +0000
Subject: [PATCH] Publish to our Repo


diff --git a/build-logic/src/main/kotlin/geyser.modded-conventions.gradle.kts b/build-logic/src/main/kotlin/geyser.modded-conventions.gradle.kts
index b75e9c5beb9e518d553415b6c9369f0674a6c88d..7a7daeaad71164253d670a501f5a57e766b7bb39 100644
--- a/build-logic/src/main/kotlin/geyser.modded-conventions.gradle.kts
+++ b/build-logic/src/main/kotlin/geyser.modded-conventions.gradle.kts
@@ -118,6 +118,7 @@ repositories {
     // mavenLocal()
     maven("https://repo.opencollab.dev/maven-releases/")
     maven("https://repo.opencollab.dev/maven-snapshots/")
+    maven("https://repo.kalimero2.com/snapshots/")
     maven("https://jitpack.io")
     maven("https://oss.sonatype.org/content/repositories/snapshots/")
     maven("https://s01.oss.sonatype.org/content/repositories/snapshots/")
diff --git a/build-logic/src/main/kotlin/geyser.publish-conventions.gradle.kts b/build-logic/src/main/kotlin/geyser.publish-conventions.gradle.kts
index eca5877216da6dc2bd3bdca95a9b8db00ce78522..4588ee52176d093530d4de716435eac07dd4631d 100644
--- a/build-logic/src/main/kotlin/geyser.publish-conventions.gradle.kts
+++ b/build-logic/src/main/kotlin/geyser.publish-conventions.gradle.kts
@@ -4,8 +4,8 @@ plugins {
 }
 
 indra {
-    publishSnapshotsTo("geysermc", "https://repo.opencollab.dev/maven-snapshots")
-    publishReleasesTo("geysermc", "https://repo.opencollab.dev/maven-releases")
+    publishSnapshotsTo("kalimero2", "https://repo.kalimero2.com/snapshots")
+    publishReleasesTo("kalimero2", "https://repo.kalimero2.com/releases")
 }
 
 publishing {
diff --git a/gradle.properties b/gradle.properties
index dd61d2556751d35487ba01931fe63497659f6f1a..ae7dd2821ebdf8aa0689e113b18f96be297271b7 100644
--- a/gradle.properties
+++ b/gradle.properties
@@ -5,7 +5,7 @@ org.gradle.parallel=true
 org.gradle.caching=true
 org.gradle.vfs.watch=false
 
-group=org.geysermc
-id=geyser
+group=com.kalimero2.team
+id=thermalquelle
 version=2.3.0-SNAPSHOT
 description=Allows for players from Minecraft: Bedrock Edition to join Minecraft: Java Edition servers.
