From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: byquanton <32410361+byquanton@users.noreply.github.com>
Date: Mon, 2 Oct 2023 11:43:24 +0000
Subject: [PATCH] Publish to our Repo


diff --git a/build-logic/src/main/kotlin/geyser.publish-conventions.gradle.kts b/build-logic/src/main/kotlin/geyser.publish-conventions.gradle.kts
index eca5877216da6dc2bd3bdca95a9b8db00ce78522..4588ee52176d093530d4de716435eac07dd4631d 100644
--- a/build-logic/src/main/kotlin/geyser.publish-conventions.gradle.kts
+++ b/build-logic/src/main/kotlin/geyser.publish-conventions.gradle.kts
@@ -4,8 +4,8 @@ plugins {
 }
 
 indra {
-    publishSnapshotsTo("geysermc", "https://repo.opencollab.dev/maven-snapshots")
-    publishReleasesTo("geysermc", "https://repo.opencollab.dev/maven-releases")
+    publishSnapshotsTo("kalimero2", "https://repo.kalimero2.com/snapshots")
+    publishReleasesTo("kalimero2", "https://repo.kalimero2.com/releases")
 }
 
 publishing {
diff --git a/gradle.properties b/gradle.properties
index a8e5eaaaf99421c780035a52e8a23f9f219ea8e1..b1a9ed8e85c2fa43f36c2167cbbf294c406f7def 100644
--- a/gradle.properties
+++ b/gradle.properties
@@ -5,7 +5,7 @@ org.gradle.parallel=true
 org.gradle.caching=true
 org.gradle.vfs.watch=false
 
-group=org.geysermc
-id=geyser
+group=com.kalimero2.team
+id=thermalquelle
 version=2.2.2-SNAPSHOT
-description=Allows for players from Minecraft: Bedrock Edition to join Minecraft: Java Edition servers.
\ No newline at end of file
+description=Allows for players from Minecraft: Bedrock Edition to join Minecraft: Java Edition servers.
