From dfe4763237dda9c8d9ff5d815a59ef4a19026308 Mon Sep 17 00:00:00 2001
From: byquanton <32410361+byquanton@users.noreply.github.com>
Date: Mon, 2 Oct 2023 11:43:24 +0000
Subject: [PATCH] Publish to our Repo


diff --git a/build-logic/src/main/kotlin/geyser.publish-conventions.gradle.kts b/build-logic/src/main/kotlin/geyser.publish-conventions.gradle.kts
index eca587721..4588ee521 100644
--- a/build-logic/src/main/kotlin/geyser.publish-conventions.gradle.kts
+++ b/build-logic/src/main/kotlin/geyser.publish-conventions.gradle.kts
@@ -4,8 +4,8 @@ plugins {
 }
 
 indra {
-    publishSnapshotsTo("geysermc", "https://repo.opencollab.dev/maven-snapshots")
-    publishReleasesTo("geysermc", "https://repo.opencollab.dev/maven-releases")
+    publishSnapshotsTo("kalimero2", "https://repo.kalimero2.com/snapshots")
+    publishReleasesTo("kalimero2", "https://repo.kalimero2.com/releases")
 }
 
 publishing {

