From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: byquanton <32410361+byquanton@users.noreply.github.com>
Date: Wed, 4 Oct 2023 21:36:52 +0200
Subject: [PATCH] Add Thermalquelle Branding


diff --git a/core/src/main/java/org/geysermc/geyser/GeyserImpl.java b/core/src/main/java/org/geysermc/geyser/GeyserImpl.java
index 8febf4d217961d374bc3c425bbd67da92a14d1b9..55915d1ab409eced07a5ac9b7c6a6b9c2fd74d40 100644
--- a/core/src/main/java/org/geysermc/geyser/GeyserImpl.java
+++ b/core/src/main/java/org/geysermc/geyser/GeyserImpl.java
@@ -138,6 +138,7 @@ public class GeyserImpl implements GeyserApi, EventRegistrar {
             .enable(JsonParser.Feature.ALLOW_SINGLE_QUOTES);
 
     public static final String NAME = "Geyser";
+    public static final String BRAND_NAME = "Thermalquelle"; // Thermalquelle - Add Brand Name
     public static final String GIT_VERSION = BuildData.GIT_VERSION;
     public static final String VERSION = BuildData.VERSION;
 
@@ -223,7 +224,7 @@ public class GeyserImpl implements GeyserApi, EventRegistrar {
 
         logger.info("******************************************");
         logger.info("");
-        logger.info(GeyserLocale.getLocaleStringLog("geyser.core.load", NAME, VERSION));
+        logger.info(GeyserLocale.getLocaleStringLog("geyser.core.load", BRAND_NAME, VERSION)); // Thermalquelle - Change to BRAND_NAME
         logger.info("");
         if (IS_DEV) {
             logger.info(GeyserLocale.getLocaleStringLog("geyser.core.dev_build", "https://discord.gg/geysermc"));
diff --git a/core/src/main/java/org/geysermc/geyser/command/defaults/VersionCommand.java b/core/src/main/java/org/geysermc/geyser/command/defaults/VersionCommand.java
index 8d34c1bf047f299d1f1f8efb548e1001674ff163..cf4000ab687e4570f3c685701f94e067840b6c54 100644
--- a/core/src/main/java/org/geysermc/geyser/command/defaults/VersionCommand.java
+++ b/core/src/main/java/org/geysermc/geyser/command/defaults/VersionCommand.java
@@ -70,8 +70,10 @@ public class VersionCommand extends GeyserCommand {
         }
 
         source.sendMessage(GeyserLocale.getPlayerLocaleString("geyser.commands.version.version", source.locale(),
-                GeyserImpl.NAME, GeyserImpl.VERSION, javaVersions, bedrockVersions));
+            GeyserImpl.BRAND_NAME, GeyserImpl.VERSION, javaVersions, bedrockVersions)); // Thermalquelle - Change to BRAND_NAME
 
+        // Thermalquelle start - remove update checking
+        /*
         // Disable update checking in dev mode and for players in Geyser Standalone
         if (!GeyserImpl.getInstance().isProductionEnvironment() || (!source.isConsole() && geyser.getPlatformType() == PlatformType.STANDALONE)) {
             return;
@@ -101,5 +103,7 @@ public class VersionCommand extends GeyserCommand {
             GeyserImpl.getInstance().getLogger().error(GeyserLocale.getLocaleStringLog("geyser.commands.version.failed"), e);
             source.sendMessage(ChatColor.RED + GeyserLocale.getPlayerLocaleString("geyser.commands.version.failed", source.locale()));
         }
+        */
+        // Thermalquelle end
     }
 }
