From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: byquanton <32410361+byquanton@users.noreply.github.com>
Date: Wed, 4 Oct 2023 21:36:52 +0200
Subject: [PATCH] Add Thermalquelle Branding


diff --git a/core/src/main/java/org/geysermc/geyser/GeyserImpl.java b/core/src/main/java/org/geysermc/geyser/GeyserImpl.java
index 8b54f5298e84e4f2df0b5329c90e695c4c17b236..995f8a485a0560593032621c5e80304a5ea1489d 100644
--- a/core/src/main/java/org/geysermc/geyser/GeyserImpl.java
+++ b/core/src/main/java/org/geysermc/geyser/GeyserImpl.java
@@ -114,6 +114,7 @@ public class GeyserImpl implements GeyserApi {
             .enable(JsonParser.Feature.ALLOW_SINGLE_QUOTES);
 
     public static final String NAME = "Geyser";
+    public static final String BRAND_NAME = "Thermalquelle"; // Thermalquelle - Add Brand Name
     public static final String GIT_VERSION = "${gitVersion}";
     public static final String VERSION = "${version}";
 
@@ -204,7 +205,7 @@ public class GeyserImpl implements GeyserApi {
 
         logger.info("******************************************");
         logger.info("");
-        logger.info(GeyserLocale.getLocaleStringLog("geyser.core.load", NAME, VERSION));
+        logger.info(GeyserLocale.getLocaleStringLog("geyser.core.load", BRAND_NAME, VERSION)); // Thermalquelle - Change to BRAND_NAME
         logger.info("");
         logger.info("******************************************");
 
diff --git a/core/src/main/java/org/geysermc/geyser/command/defaults/VersionCommand.java b/core/src/main/java/org/geysermc/geyser/command/defaults/VersionCommand.java
index eb2e8ff47bdc9b555900cc3c48b488fa7d6aa943..afb35c74e5ba9cdbd26213eb60749a2dbb2ae15f 100644
--- a/core/src/main/java/org/geysermc/geyser/command/defaults/VersionCommand.java
+++ b/core/src/main/java/org/geysermc/geyser/command/defaults/VersionCommand.java
@@ -69,8 +69,10 @@ public class VersionCommand extends GeyserCommand {
         }
 
         sender.sendMessage(GeyserLocale.getPlayerLocaleString("geyser.commands.version.version", sender.locale(),
-                GeyserImpl.NAME, GeyserImpl.VERSION, javaVersions, bedrockVersions));
+                GeyserImpl.BRAND_NAME, GeyserImpl.VERSION, javaVersions, bedrockVersions)); // Thermalquelle - Change to BRAND_NAME
 
+        // Thermalquelle start - remove update checking
+        /*
         // Disable update checking in dev mode and for players in Geyser Standalone
         if (GeyserImpl.getInstance().isProductionEnvironment() && !(!sender.isConsole() && geyser.getPlatformType() == PlatformType.STANDALONE)) {
             sender.sendMessage(GeyserLocale.getPlayerLocaleString("geyser.commands.version.checking", sender.locale()));
@@ -94,6 +96,8 @@ public class VersionCommand extends GeyserCommand {
                 sender.sendMessage(ChatColor.RED + GeyserLocale.getPlayerLocaleString("geyser.commands.version.failed", sender.locale()));
             }
         }
+        */
+        // Thermalquelle end
     }
 
     @Override
