From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: byquanton <32410361+byquanton@users.noreply.github.com>
Date: Sun, 15 Oct 2023 15:31:47 +0200
Subject: [PATCH] Fix Legacy Formatting Issue with Item Names


diff --git a/core/src/main/java/org/geysermc/geyser/translator/text/MessageTranslator.java b/core/src/main/java/org/geysermc/geyser/translator/text/MessageTranslator.java
index 5f811ab4939e2dc030e936f24443b595f179d8f6..2c04135a3710b974a6c63e7055d227e14362c4cd 100644
--- a/core/src/main/java/org/geysermc/geyser/translator/text/MessageTranslator.java
+++ b/core/src/main/java/org/geysermc/geyser/translator/text/MessageTranslator.java
@@ -39,8 +39,10 @@ import net.kyori.adventure.text.serializer.legacy.LegacyComponentSerializer;
 import net.kyori.adventure.text.serializer.plain.PlainTextComponentSerializer;
 import org.cloudburstmc.protocol.bedrock.packet.TextPacket;
 import org.geysermc.geyser.GeyserImpl;
+import org.geysermc.geyser.GeyserLogger;
 import org.geysermc.geyser.session.GeyserSession;
 import org.geysermc.geyser.text.*;
+import org.jetbrains.annotations.NotNull;
 
 import java.util.*;
 
@@ -215,19 +217,28 @@ public class MessageTranslator {
             return message;
         }
 
-        try {
-            return convertJsonMessage(message, locale);
-        } catch (Exception ignored) {
-            // Use the default legacy serializer since message is java-legacy
-            String convertedMessage = convertMessage(LegacyComponentSerializer.legacySection().deserialize(message), locale);
-
-            // We have to do this since Adventure strips the starting reset character
-            if (message.startsWith(RESET) && !convertedMessage.startsWith(RESET)) {
-                convertedMessage = RESET + convertedMessage;
+        if(!message.contains("§")) {
+            try {
+                return convertJsonMessage(message, locale);
+            } catch (Exception e) {
+                return convertLegacyMessage(message, locale);
             }
+        } else {
+            return convertLegacyMessage(message, locale);
+        }
+    }
+
+    @NotNull
+    private static String convertLegacyMessage(String message, String locale) {
+        // Use the default legacy serializer since message is java-legacy
+        String convertedMessage = convertMessage(LegacyComponentSerializer.legacySection().deserialize(message), locale);
 
-            return convertedMessage;
+        // We have to do this since Adventure strips the starting reset character
+        if (message.startsWith(RESET) && !convertedMessage.startsWith(RESET)) {
+            convertedMessage = RESET + convertedMessage;
         }
+
+        return convertedMessage;
     }
 
     public static String convertMessageLenient(String message) {
