From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: byquanton <32410361+byquanton@users.noreply.github.com>
Date: Wed, 4 Oct 2023 21:40:17 +0200
Subject: [PATCH] Always send that the resource pack was loaded


diff --git a/core/src/main/java/org/geysermc/geyser/translator/protocol/java/JavaClientboundResourcePacksPacket.java b/core/src/main/java/org/geysermc/geyser/translator/protocol/java/JavaClientboundResourcePacksPacket.java
index 01750228093f1f684df4c952e314710b414d8bca..96176f025ce126daec3525b30e372411ecc05694 100644
--- a/core/src/main/java/org/geysermc/geyser/translator/protocol/java/JavaClientboundResourcePacksPacket.java
+++ b/core/src/main/java/org/geysermc/geyser/translator/protocol/java/JavaClientboundResourcePacksPacket.java
@@ -38,6 +38,6 @@ public class JavaClientboundResourcePacksPacket extends PacketTranslator<Clientb
     @Override
     public void translate(GeyserSession session, ClientboundResourcePackPacket packet) {
         // We need to "answer" this to avoid timeout issues related to resource packs
-        session.sendDownstreamPacket(new ServerboundResourcePackPacket(ResourcePackStatus.DECLINED));
+        session.sendDownstreamPacket(new ServerboundResourcePackPacket(ResourcePackStatus.SUCCESSFULLY_LOADED)); // Thermalquelle - Change to SUCCESSFULLY_LOADED
     }
 }
